### ISSUE TITLE
`<Tree.Item>` actions use context menu

### ISSUE THREAD

**Issue by FlyersPh9** [iTwin/stratakit#617]
Improvement to https://github.com/iTwin/design-system/issues/335

The actions should be additionally exposed from a custom [context menu](https://developer.mozilla.org/en-US/docs/Web/API/Element/contextmenu_event) on the TreeItem. This will benefit both mouse users (right click) and keyboard users (Shift+F10 on Windows).
